avocado/plugins/runner_nrunner.py

```diff
@@ -72,9 +72,9 @@ def _populate_task_logdir(self, base_path, task, statuses, debug=False):
         with open(data_file, 'w') as fp:
             fp.write("{}\n".format(task.output_dir))
 
-    def run_suite(self, job, result, test_suite, variants, timeout=0,
-                  replay_map=None, execution_order=None):
+    def run_suite(self, job, result, test_suite, variants, config):
         summary = set()
+        timeout = config.get('run.job_timeout')
```

Comment:
Don't we need to check the time format, just as you did on `avocado/plugins/runner.py`?

`timeout = data_structures.time_to_seconds(config.get('run.job_timeout'))`

Reply:
Nice catch, thanks.

But after a better thinking, maybe the best approach is to register_option() with `key_type=time_to_seconds`. Will fix on a v2.